### DISCUSSION TITLE
Failed to verify the root directory of local source -- NuGet NU1301

### DISCUSSION THREAD

**Discussion by jose-vitor-quadrado** [balta-io/forum#172] [Tira Dúvidas] [answered]
### Não estou conseguindo utilizar nenhum pacote nuget!

![nuget-err](https://user-images.githubusercontent.com/66926989/207952471-b8243c2d-97d0-468f-b095-320bb810ce16.png)
Este mesmo erro acontece quando tento instalar qualquer pacote NuGet

**Comment by DiegoOliver94** [comment, marked as answer]
Este mesmo erro aconteceu comigo, como utilizo o notebook da empresa ele referenciou o nuget deles, nesse caso consegui resolver quando inseri o --version (numero da versao) ao instalar um pacote, exemplo:

dotnet add package Dapper --version 2.0.123

também pode usar o site para saber qual versão atual ou condizente com a versão do .net esta usando: https://www.nuget.org/packages/Dapper

**Reply by jose-vitor-quadrado** [reply]
Boa tade, no print em questão eu já estava utilizando o comando com a versão inclusive está amostra na imagem, neste caso pra resolver tive que apagar o source que havia criado para conseguir instalar os pacotes!
